### ISSUE TITLE
Empty ocrHighlighting snippets (n:1)

### ISSUE THREAD

**Issue by paroar** [dbmdz/solr-ocrhighlighting#173]
Hi,

I have indexed multiple files per document (n:1) and the number of snippets for a document that should be 24 returns empty.

Request 24:

```sh
curl --location --request GET 'http://localhost:8983/solr/ocr/select?defType=edismax&hl.snippets=24&hl.weightMatches=true&hl=on&fl=id,ocr_text&qf=ocr_text&hl.ocr.fl=ocr_text&q=ocr_text:gallega%20AND%20id:es-scbg_bblg_18950101'
```

Response 24:

```json
{
    "responseHeader": {
        "status": 0,
        "QTime": 36
    },
    "response": {
        "numFound": 1,
        "start": 0,
        "numFoundExact": true,
        "docs": [
            {
                "id": "es-scbg_bblg_18950101",
                "ocr_text": "/altos/volume1/es-scbg_bblg_18950101/es-scbg_bblg_18950101_0008.xml+/altos/volume1/es-scbg_bblg_18950101/es-scbg_bblg_18950101_0003.xml+/altos/volume1/es-scbg_bblg_18950101/es-scbg_bblg_18950101_0002.xml+/altos/volume1/es-scbg_bblg_18950101/es-scbg_bblg_18950101_0001.xml+/altos/volume1/es-scbg_bblg_18950101/es-scbg_bblg_18950101_0005.xml+/altos/volume1/es-scbg_bblg_18950101/es-scbg_bblg_18950101_0004.xml+/altos/volume1/es-scbg_bblg_18950101/es-scbg_bblg_18950101_0006.xml+/altos/volume1/es-scbg_bblg_18950101/es-scbg_bblg_18950101_0007.xml"
            }
        ]
    },
    "ocrHighlighting": {
        "es-scbg_bblg_18950101": {}
    }
}
```

The thing is that if I reduce the number of snippets to 2 I get 2 snippets in the response.

Request 2:

```sh
curl --location --request GET 'http://localhost:8983/solr/ocr/select?defType=edismax&hl.snippets=2&hl.weightMatches=true&hl=on&fl=id,ocr_text&qf=ocr_text&hl.ocr.fl=ocr_text&q=ocr_text:gallega%20AND%20id:es-scbg_bblg_18950101'
```

Response 2:

```json
{
    "responseHeader": {
        "status": 0,
        "QTime": 21
    },
    "response": {
        "numFound": 1,
        "start": 0,
        "numFoundExact": true,
        "docs": [
            {
                "id": "es-scbg_bblg_18950101",
                "ocr_text": "/altos/volume1/es-scbg_bblg_18950101/es-scbg_bblg_18950101_0008.xml+/altos/volume1/es-scbg_bblg_18950101/es-scbg_bblg_18950101_0003.xml+/altos/volume1/es-scbg_bblg_18950101/es-scbg_bblg_18950101_0002.xml+/altos/volume1/es-scbg_bblg_18950101/es-scbg_bblg_18950101_0001.xml+/altos/volume1/es-scbg_bblg_18950101/es-scbg_bblg_18950101_0005.xml+/altos/volume1/es-scbg_bblg_18950101/es-scbg_bblg_18950101_0004.xml+/altos/volume1/es-scbg_bblg_18950101/es-scbg_bblg_18950101_0006.xml+/altos/volume1/es-scbg_bblg_18950101/es-scbg_bblg_18950101_0007.xml"
            }
        ]
    },
    "ocrHighlighting": {
        "es-scbg_bblg_18950101": {
            "ocr_text": {
                "snippets": [
                    {
                        "text": "DE LA LIBRERIA <em>GALLEGA</em>.",
                        "score": 1142045.4,
                        "pages": [
                            {
                                "id": "Page7",
                                "width": 1363,
                                "height": 1938
                            }
                        ],
                        "regions": [
                            {
                                "ulx": 452,
                                "uly": 90,
                                "lrx": 912,
                                "lry": 116,
                                "text": "DE LA LIBRERIA <em>GALLEGA</em>.",
                                "pageIdx": 0
                            }
                        ],
                        "highlights": [
                            [
                                {
                                    "ulx": 312,
                                    "uly": 5,
                                    "lrx": 460,
                                    "lry": 26,
                                    "text": "GALLEGA.",
                                    "parentRegionIdx": 0
                                }
                            ]
                        ]
                    },
                    {
                        "text": "DE LA LIBRERIA <em>GALLEGA</em>.",
                        "score": 825861.7,
                        "pages": [
                            {
                                "id": "Page5",
                                "width": 1365,
                                "height": 1935
                            }
                        ],
                        "regions": [
                            {
                                "ulx": 468,
                                "uly": 68,
                                "lrx": 927,
                                "lry": 93,
                                "text": "DE LA LIBRERIA <em>GALLEGA</em>.",
                                "pageIdx": 0
                            }
                        ],
                        "highlights": [
                            [
                                {
                                    "ulx": 313,
                                    "uly": 0,
                                    "lrx": 459,
                                    "lry": 25,
                                    "text": "GALLEGA.",
                                    "parentRegionIdx": 0
                                }
                            ]
                        ]
                    }
                ],
                "numTotal": 24
            }
        }
    }
}
```

The empty snippets starts at 9, until then everything seems to work fine.

Request 9:

```sh
curl --location --request GET 'http://localhost:8983/solr/ocr/select?defType=edismax&hl.snippets=9&hl.weightMatches=true&hl=on&fl=id,ocr_text&qf=ocr_text&hl.ocr.fl=ocr_text&q=ocr_text:gallega%20AND%20id:es-scbg_bblg_18950101'
````
Response 9:

```json
{
    "responseHeader": {
        "status": 0,
        "QTime": 33
    },
    "response": {
        "numFound": 1,
        "start": 0,
        "numFoundExact": true,
        "docs": [
            {
                "id": "es-scbg_bblg_18950101",
                "ocr_text": "/altos/volume1/es-scbg_bblg_18950101/es-scbg_bblg_18950101_0008.xml+/altos/volume1/es-scbg_bblg_18950101/es-scbg_bblg_18950101_0003.xml+/altos/volume1/es-scbg_bblg_18950101/es-scbg_bblg_18950101_0002.xml+/altos/volume1/es-scbg_bblg_18950101/es-scbg_bblg_18950101_0001.xml+/altos/volume1/es-scbg_bblg_18950101/es-scbg_bblg_18950101_0005.xml+/altos/volume1/es-scbg_bblg_18950101/es-scbg_bblg_18950101_0004.xml+/altos/volume1/es-scbg_bblg_18950101/es-scbg_bblg_18950101_0006.xml+/altos/volume1/es-scbg_bblg_18950101/es-scbg_bblg_18950101_0007.xml"
            }
        ]
    },
    "ocrHighlighting": {
        "es-scbg_bblg_18950101": {}
    }
}
```

I tried to sort the ocr_text so the pages are in order, but the results are not the same (goes up to 18 instead of 9) but similar, so the order matters.

Thanks!

**Comment by jbaiter** [comment]
- Can you check the Solr log for a stack trace that might tell us more about the cause of the problem?
- Can you share the files the problem occurs with?

**Comment by paroar** [comment]
Sure, we are using the 0.6.0 version with Solr 8.8 if that helps.

Solr log
```log
Executing /opt/docker-solr/scripts/solr-precreate ocr /var/solr/data/core-config
Executing /opt/docker-solr/scripts/precreate-core ocr /var/solr/data/core-config
Created ocr
Starting Solr 
OpenJDK 64-Bit Server VM warning: Failed to reserve shared memory. (error = 1)
OpenJDK 64-Bit Server VM warning: Failed to reserve shared memory. (error = 1)
OpenJDK 64-Bit Server VM warning: Failed to reserve shared memory. (error = 1)
2021-06-11 07:27:51.534 INFO  (main) [   ] o.e.j.u.log Logging initialized @1362ms to org.eclipse.jetty.util.log.Slf4jLog
2021-06-11 07:27:51.798 INFO  (main) [   ] o.e.j.s.Server jetty-9.4.34.v20201102; built: 2020-11-02T14:15:39.302Z; git: e46af88704a893fc12cb0e3bf46e2c7b48a009e7; jvm 11.0.11+9
2021-06-11 07:27:51.850 INFO  (main) [   ] o.e.j.d.p.ScanningAppProvider Deployment monitor [file:///opt/solr-8.8.2/server/contexts/] at interval 0
2021-06-11 07:27:52.185 INFO  (main) [   ] o.e.j.w.StandardDescriptorProcessor NO JSP Support for /solr, did not find org.apache.jasper.servlet.JspServlet
2021-06-11 07:27:52.204 INFO  (main) [   ] o.e.j.s.session DefaultSessionIdManager workerName=node0
2021-06-11 07:27:52.204 INFO  (main) [   ] o.e.j.s.session No SessionScavenger set, using defaults
2021-06-11 07:27:52.207 INFO  (main) [   ] o.e.j.s.session node0 Scavenging every 600000ms
2021-06-11 07:27:52.306 INFO  (main) [   ] o.a.s.s.SolrDispatchFilter Using logger factory org.apache.logging.slf4j.Log4jLoggerFactory
2021-06-11 07:27:52.311 INFO  (main) [   ] o.a.s.s.SolrDispatchFilter  ___      _       Welcome to Apache Solr™ version 8.8.2
2021-06-11 07:27:52.312 INFO  (main) [   ] o.a.s.s.SolrDispatchFilter / __| ___| |_ _   Starting in standalone mode on port 8983
2021-06-11 07:27:52.312 INFO  (main) [   ] o.a.s.s.SolrDispatchFilter \__ \/ _ \ | '_|  Install dir: /opt/solr
2021-06-11 07:27:52.312 INFO  (main) [   ] o.a.s.s.SolrDispatchFilter |___/\___/_|_|    Start time: 2021-06-11T07:27:52.312615Z
2021-06-11 07:27:52.356 INFO  (main) [   ] o.a.s.c.SolrPaths Using system property solr.solr.home: /var/solr/data
2021-06-11 07:27:52.358 INFO  (main) [   ] o.a.s.c.SolrXmlConfig Loading container configuration from /var/solr/data/solr.xml
2021-06-11 07:27:52.468 INFO  (main) [   ] o.a.s.c.SolrXmlConfig MBean server found: com.sun.jmx.mbeanserver.JmxMBeanServer@4b29d1d2, but no JMX reporters were configured - adding default JMX reporter.
2021-06-11 07:27:53.132 INFO  (main) [   ] o.a.s.h.c.HttpShardHandlerFactory Host whitelist initialized: WhitelistHostChecker [whitelistHosts=null, whitelistHostCheckingEnabled=true]
2021-06-11 07:27:53.351 WARN  (main) [   ] o.e.j.u.s.S.config Trusting all certificates configured for Client@6c1832aa[provider=null,keyStore=null,trustStore=null]
2021-06-11 07:27:53.351 WARN  (main) [   ] o.e.j.u.s.S.config No Client EndPointIdentificationAlgorithm configured for Client@6c1832aa[provider=null,keyStore=null,trustStore=null]
2021-06-11 07:27:53.620 WARN  (main) [   ] o.e.j.u.s.S.config Trusting all certificates configured for Client@74294c1a[provider=null,keyStore=null,trustStore=null]
2021-06-11 07:27:53.620 WARN  (main) [   ] o.e.j.u.s.S.config No Client EndPointIdentificationAlgorithm configured for Client@74294c1a[provider=null,keyStore=null,trustStore=null]
2021-06-11 07:27:53.686 WARN  (main) [   ] o.a.s.c.CoreContainer Not all security plugins configured!  authentication=disabled authorization=disabled.  Solr is only as secure as you make it. Consider configuring authentication/authorization before exposing Solr to users internal or external.  See https://s.apache.org/solrsecurity for more info
2021-06-11 07:27:53.904 INFO  (main) [   ] o.a.s.c.TransientSolrCoreCacheDefault Allocating transient cache for 2147483647 transient cores
2021-06-11 07:27:53.907 INFO  (main) [   ] o.a.s.h.a.MetricsHistoryHandler No .system collection, keeping metrics history in memory.
2021-06-11 07:27:54.044 INFO  (main) [   ] o.a.s.m.r.SolrJmxReporter JMX monitoring for 'solr.node' (registry 'solr.node') enabled at server: com.sun.jmx.mbeanserver.JmxMBeanServer@4b29d1d2
2021-06-11 07:27:54.045 INFO  (main) [   ] o.a.s.m.r.SolrJmxReporter JMX monitoring for 'solr.jvm' (registry 'solr.jvm') enabled at server: com.sun.jmx.mbeanserver.JmxMBeanServer@4b29d1d2
2021-06-11 07:27:54.051 INFO  (main) [   ] o.a.s.m.r.SolrJmxReporter JMX monitoring for 'solr.jetty' (registry 'solr.jetty') enabled at server: com.sun.jmx.mbeanserver.JmxMBeanServer@4b29d1d2
2021-06-11 07:27:54.109 INFO  (main) [   ] o.a.s.c.CorePropertiesLocator Found 1 core definitions underneath /var/solr/data
2021-06-11 07:27:54.111 INFO  (main) [   ] o.a.s.c.CorePropertiesLocator Cores are: [ocr]
2021-06-11 07:27:54.206 INFO  (coreLoadExecutor-13-thread-1) [   x:ocr] o.a.s.c.SolrResourceLoader Added 1 libs to classloader, from paths: [/var/solr/data/plugins]
2021-06-11 07:27:54.287 INFO  (main) [   ] o.e.j.s.h.ContextHandler Started o.e.j.w.WebAppContext@1e1d3956{/solr,file:///opt/solr-8.8.2/server/solr-webapp/webapp/,AVAILABLE}{/opt/solr-8.8.2/server/solr-webapp/webapp}
2021-06-11 07:27:54.310 INFO  (main) [   ] o.e.j.s.AbstractConnector Started ServerConnector@62e20a76{HTTP/1.1, (http/1.1, h2c)}{0.0.0.0:8983}
2021-06-11 07:27:54.311 INFO  (main) [   ] o.e.j.s.Server Started @4145ms
2021-06-11 07:27:54.492 INFO  (coreLoadExecutor-13-thread-1) [   x:ocr] o.a.s.c.SolrConfig Using Lucene MatchVersion: 8.8.2
2021-06-11 07:27:54.586 INFO  (coreLoadExecutor-13-thread-1) [   x:ocr] o.a.s.s.IndexSchema Schema name=coordinateHighlight
2021-06-11 07:27:54.666 INFO  (coreLoadExecutor-13-thread-1) [   x:ocr] o.a.s.s.IndexSchema Loaded schema coordinateHighlight/1.0 with uniqueid field id
2021-06-11 07:27:54.672 INFO  (coreLoadExecutor-13-thread-1) [   x:ocr] o.a.s.c.CoreContainer Creating SolrCore 'ocr' using configuration from instancedir /var/solr/data/ocr, trusted=true
2021-06-11 07:27:54.721 INFO  (coreLoadExecutor-13-thread-1) [   x:ocr] o.a.s.m.r.SolrJmxReporter JMX monitoring for 'solr.core.ocr' (registry 'solr.core.ocr') enabled at server: com.sun.jmx.mbeanserver.JmxMBeanServer@4b29d1d2
2021-06-11 07:27:54.741 INFO  (coreLoadExecutor-13-thread-1) [   x:ocr] o.a.s.c.SolrCore [[ocr] ] Opening new SolrCore at [/var/solr/data/ocr], dataDir=[/var/solr/data/ocr/data/]
2021-06-11 07:27:55.305 INFO  (coreLoadExecutor-13-thread-1) [   x:ocr] o.a.s.u.CommitTracker Hard AutoCommit: disabled
2021-06-11 07:27:55.306 INFO  (coreLoadExecutor-13-thread-1) [   x:ocr] o.a.s.u.CommitTracker Soft AutoCommit: disabled
2021-06-11 07:27:55.348 WARN  (coreLoadExecutor-13-thread-1) [   x:ocr] o.a.s.r.ManagedResourceStorage Cannot write to config directory /var/solr/data/ocr/conf ; switching to use InMemory storage instead.
2021-06-11 07:27:55.378 INFO  (coreLoadExecutor-13-thread-1) [   x:ocr] o.a.s.h.ReplicationHandler Commits will be reserved for 10000 ms
2021-06-11 07:27:55.396 INFO  (searcherExecutor-15-thread-1-processing-x:ocr) [   x:ocr] o.a.s.c.SolrCore [ocr]  Registered new searcher autowarm time: 0 ms
2021-06-11 07:28:03.917 INFO  (searcherExecutor-15-thread-1-processing-x:ocr) [   x:ocr] o.a.s.c.SolrCore [ocr]  Registered new searcher autowarm time: 1 ms
2021-06-11 07:28:03.922 INFO  (qtp722951168-20) [   x:ocr] o.a.s.u.p.LogUpdateProcessorFactory [ocr]  webapp=/solr path=/update/json/docs params={commit=true}{add=[es-scbg_bblg_18950101],commit=} 0 660
2021-06-11 07:28:42.209 ERROR (qtp722951168-15) [   x:ocr] d.d.s.l.OcrHighlighter Could not highlight OCR content for document => java.lang.IllegalArgumentException: Invalid range: [2829..2828)
	at com.google.common.collect.Range.<init>(Range.java:352)
java.lang.IllegalArgumentException: Invalid range: [2829..2828)
	at com.google.common.collect.Range.<init>(Range.java:352) ~[?:?]
	at com.google.common.collect.Range.create(Range.java:155) ~[?:?]
	at com.google.common.collect.Range.closedOpen(Range.java:189) ~[?:?]
	at de.digitalcollections.solrocr.formats.alto.AltoFormat.getContainingWordLimits(AltoFormat.java:122) ~[?:?]
	at de.digitalcollections.solrocr.lucene.OcrPassageFormatter.adjustPositionToCharacterEntities(OcrPassageFormatter.java:172) ~[?:?]
	at de.digitalcollections.solrocr.lucene.OcrPassageFormatter.getHighlightedFragment(OcrPassageFormatter.java:155) ~[?:?]
	at de.digitalcollections.solrocr.lucene.OcrPassageFormatter.format(OcrPassageFormatter.java:191) ~[?:?]
	at de.digitalcollections.solrocr.lucene.OcrPassageFormatter.format(OcrPassageFormatter.java:100) ~[?:?]
	at de.digitalcollections.solrocr.lucene.OcrFieldHighlighter.highlightFieldForDoc(OcrFieldHighlighter.java:104) ~[?:?]
	at de.digitalcollections.solrocr.lucene.OcrHighlighter.highlightOcrFields(OcrHighlighter.java:373) ~[?:?]
	at de.digitalcollections.solrocr.solr.SolrOcrHighlighter.doHighlighting(SolrOcrHighlighter.java:73) ~[?:?]
	at de.digitalcollections.solrocr.solr.OcrHighlightComponent.process(OcrHighlightComponent.java:121) ~[?:?]
	at org.apache.solr.handler.component.SearchHandler.handleRequestBody(SearchHandler.java:355) ~[?:?]
	at org.apache.solr.handler.RequestHandlerBase.handleRequest(RequestHandlerBase.java:216) ~[?:?]
	at org.apache.solr.core.SolrCore.execute(SolrCore.java:2646) ~[?:?]
	at org.apache.solr.servlet.HttpSolrCall.execute(HttpSolrCall.java:794) ~[?:?]
	at org.apache.solr.servlet.HttpSolrCall.call(HttpSolrCall.java:567) ~[?:?]
	at org.apache.solr.servlet.SolrDispatchFilter.doFilter(SolrDispatchFilter.java:427) ~[?:?]
	at org.apache.solr.servlet.SolrDispatchFilter.doFilter(SolrDispatchFilter.java:357) ~[?:?]
	at org.eclipse.jetty.servlet.FilterHolder.doFilter(FilterHolder.java:201) ~[jetty-servlet-9.4.34.v20201102.jar:9.4.34.v20201102]
	at org.eclipse.jetty.servlet.ServletHandler$Chain.doFilter(ServletHandler.java:1601) ~[jetty-servlet-9.4.34.v20201102.jar:9.4.34.v20201102]
	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:548) ~[jetty-servlet-9.4.34.v20201102.jar:9.4.34.v20201102]
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:143) ~[jetty-server-9.4.34.v20201102.jar:9.4.34.v20201102]
	at org.eclipse.jetty.security.SecurityHandler.handle(SecurityHandler.java:602) ~[jetty-security-9.4.34.v20201102.jar:9.4.34.v20201102]
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127) ~[jetty-server-9.4.34.v20201102.jar:9.4.34.v20201102]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:235) ~[jetty-server-9.4.34.v20201102.jar:9.4.34.v20201102]
	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:1612) ~[jetty-server-9.4.34.v20201102.jar:9.4.34.v20201102]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:233) ~[jetty-server-9.4.34.v20201102.jar:9.4.34.v20201102]
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1434) ~[jetty-server-9.4.34.v20201102.jar:9.4.34.v20201102]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:188) ~[jetty-server-9.4.34.v20201102.jar:9.4.34.v20201102]
	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:501) ~[jetty-servlet-9.4.34.v20201102.jar:9.4.34.v20201102]
	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:1582) ~[jetty-server-9.4.34.v20201102.jar:9.4.34.v20201102]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:186) ~[jetty-server-9.4.34.v20201102.jar:9.4.34.v20201102]
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1349) ~[jetty-server-9.4.34.v20201102.jar:9.4.34.v20201102]
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141) ~[jetty-server-9.4.34.v20201102.jar:9.4.34.v20201102]
	at org.eclipse.jetty.server.handler.ContextHandlerCollection.handle(ContextHandlerCollection.java:191) ~[jetty-server-9.4.34.v20201102.jar:9.4.34.v20201102]
	at org.eclipse.jetty.server.handler.InetAccessHandler.handle(InetAccessHandler.java:177) ~[jetty-server-9.4.34.v20201102.jar:9.4.34.v20201102]
	at org.eclipse.jetty.server.handler.HandlerCollection.handle(HandlerCollection.java:146) ~[jetty-server-9.4.34.v20201102.jar:9.4.34.v20201102]
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127) ~[jetty-server-9.4.34.v20201102.jar:9.4.34.v20201102]
	at org.eclipse.jetty.rewrite.handler.RewriteHandler.handle(RewriteHandler.java:322) ~[jetty-rewrite-9.4.34.v20201102.jar:9.4.34.v20201102]
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127) ~[jetty-server-9.4.34.v20201102.jar:9.4.34.v20201102]
	at org.eclipse.jetty.server.Server.handle(Server.java:516) ~[jetty-server-9.4.34.v20201102.jar:9.4.34.v20201102]
	at org.eclipse.jetty.server.HttpChannel.lambda$handle$1(HttpChannel.java:383) ~[jetty-server-9.4.34.v20201102.jar:9.4.34.v20201102]
	at org.eclipse.jetty.server.HttpChannel.dispatch(HttpChannel.java:556) ~[jetty-server-9.4.34.v20201102.jar:9.4.34.v20201102]
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:375) ~[jetty-server-9.4.34.v20201102.jar:9.4.34.v20201102]
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:273) ~[jetty-server-9.4.34.v20201102.jar:9.4.34.v20201102]
	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:311) ~[jetty-io-9.4.34.v20201102.jar:9.4.34.v20201102]
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:105) ~[jetty-io-9.4.34.v20201102.jar:9.4.34.v20201102]
	at org.eclipse.jetty.io.ChannelEndPoint$1.run(ChannelEndPoint.java:104) ~[jetty-io-9.4.34.v20201102.jar:9.4.34.v20201102]
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:336) ~[jetty-util-9.4.34.v20201102.jar:9.4.34.v20201102]
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:313) ~[jetty-util-9.4.34.v20201102.jar:9.4.34.v20201102]
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:171) ~[jetty-util-9.4.34.v20201102.jar:9.4.34.v20201102]
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produce(EatWhatYouKill.java:135) ~[jetty-util-9.4.34.v20201102.jar:9.4.34.v20201102]
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:773) ~[jetty-util-9.4.34.v20201102.jar:9.4.34.v20201102]
	at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:905) ~[jetty-util-9.4.34.v20201102.jar:9.4.34.v20201102]
	at java.lang.Thread.run(Unknown Source) [?:?]
2021-06-11 07:28:42.214 INFO  (qtp722951168-15) [   x:ocr] o.a.s.c.S.Request [ocr]  webapp=/solr path=/select params={hl.snippets=24&hl.weightMatches=true&q=ocr_text:gallega+AND+id:es-scbg_bblg_18950101&defType=edismax&hl=on&qf=ocr_text&fl=id,ocr_text&hl.ocr.fl=ocr_text} hits=1 status=0 QTime=330
```

Alto files
[es-scbg_bblg_18950101.zip](https://github.com/dbmdz/solr-ocrhighlighting/files/6639309/es-scbg_bblg_18950101.zip)
